packages/import-notation/test/stringify.test.js

```diff
@@ -66,6 +66,23 @@ describe('block', () => {
             { block : 'popup', mod : { name : 'autoclosable', val : 'yes' } }
         ])).to.equal('b:popup m:theme=normal|action m:autoclosable=yes');
     });
+
+    it('should not duplicate entities', () => {
+        expect(s([
+            { block : 'popup' },
+            { block : 'popup' },
+            { block : 'popup', mod : { name : 'theme' } },
```

Comment:
Why don't we check `elem` dups?

Reply:
cause actually they are same
but I could do it for your pleasure

Reply:
nice!